src/calendar-list/index.js

```diff
@@ -139,7 +139,7 @@ class CalendarList extends Component {
     this.listView.scrollToOffset({offset: scrollAmount, animated: false});
   }
 
-  componentWillReceiveProps(props) {
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
probably not a good idea

Reply:
why?

Reply:
This method should be replaced as it's being deprecated soon